app/src/App.js

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+
+import * as apiClient from "./apiClient";
+
+const App = () => {
+  const [tasks, setTasks] = React.useState([]);
+
+  const today = new Date();
+  const [date, setDate] = React.useState(today);
+  const loadTasks = async () => setTasks(await apiClient.getTasks());
+
+  React.useEffect(() => {
+    loadTasks();
+  }, []);
+
+  const formatDate = () => {
+    const daysOfWeek = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+    ];
+    let day = daysOfWeek[date.getDay()];
+    let dd = String(date.getDate()).padStart(2, "0");
+    let mm = String(date.getMonth() + 1).padStart(2, "0");
+    let yyyy = date.getFullYear();
+
+    let dateValue = day + " " + mm + "/" + dd + "/" + yyyy;
+    return dateValue;
+  };
+
+  const changeDateValue = ({ addValue }) => {
+    setDate(date.setDate(date.getDate() + addValue));
+  };
+
+  return (
+    <main className="App">
+      <nav>
+        <h1>Lunar Task</h1>
+      </nav>
+      <section>
+        <h2>{formatDate()}</h2>
```

Comment:
I would put arrow buttons on either side of this line. The left error will modify the current state by adding -1, the right arrow would add 1 to the current date. Perhaps this also triggers the loadTasks function which could accept a parameter of "dateValue", you pass this the date for the associated journal entries. This way your tasks are always matching the current date.